tests/testdata/test-cdc/cassandra-datacenter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: cassandra.datastax.com/v1beta1
+kind: CassandraDatacenter
+metadata:
+  name: dc1
+spec:
+  clusterName: test-cluster
+  # The number of server nodes.
+  size: 1
+  cdc:
+    enabled: true
+    pulsarServiceUrl: pulsar://pulsar-proxy.pulsar.svc.cluster.local:6650
+    topicPrefix: persistent://public/default/events-
+    cdcWorkingDir: /var/lib/cassandra/cdc
```

Comment:
This is something I forgot to ask.. is this something that can be created with additionalVolumes and make it work?

Reply:
You mean create the additional volume automatically depending on what you specify for `cdcWorkingDir`? You probably could, but I don't think it is idiomatic.